packages/interactor/src/constructor.ts

```diff
@@ -152,7 +152,9 @@ export function instantiateBaseInteractor<E extends Element, F extends Filters<E
       let filter = new Filter(options.specification, filters);
       return check(`${this.description} matches filters: ${filter.description}`, () => {
         return converge(() => {
-          let element = resolver(options);
+          let lookupFilter = Filter.copy(options.filter)
+          lookupFilter.overrideFilterValues(filters)
```

Comment:
Rather than have a static function that copies, and then we mutate it. Can we just have a single, module-private function that copies and overrides in one go?